src/game.js

```diff
@@ -40,12 +40,33 @@ export class Game
             // in a full game this should be handled gracefully.
             PIXI.loader.resources['bounce'].sound.volume = value;
         });
+        testApp.state.musicVolume.subscribe(result => 
+        {
+            console.log('musicVolume: ', result);
+        });
+        testApp.state.voVolume.subscribe(result => 
+        {
+            console.log('voVolume: ', result);
+        });
 
         // add a extra state property for storying the current scene. Whenever the scene is changed, this class
         // will swap out the container attached to the stage
         this.app.state.scene = new Property(null);
         this.app.state.scene.subscribe(this.onChangeScene.bind(this));
 
+        this.application.state.musicVolume.subscribe(result => 
```

Comment:
Looks like you have some duplicate lines in here.
The sfx state is set up on lines 37-42. I'd put the music/vo lines below that. 

Also of note its `this.app.state...` not `this.application.state...`